benchmarks/dynamo/common.py

```diff
@@ -1837,6 +1837,7 @@ def run(runner, args, original_dir=None):
     if args.dynamic_shapes:
         torch._dynamo.config.dynamic_shapes = True
         torch._functorch.config.use_dynamic_shapes = True
+        torch._inductor.config.dynamic_shapes = True
```

Comment:
You're going to wanna update dynamic=True implementation too which monkeypatches these configs

Reply:
hmm... that is leaking Inductor details up into Dynamo though. I think we just need to refactor this to have Inductor pull this flag down from Dynamo. cc: @voznesenskym